CheckDrive.Api/CheckDrive.Application/DTOs/Identity/RegisterDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CheckDrive.Domain.Enums;
+using System.ComponentModel.DataAnnotations;
+
+namespace CheckDrive.Application.DTOs.Identity;
+
+public record RegisterDto(
+    string FirstName,
+    string LastName,
+    string Passport,
+    string Address,
+    string Email,
+    string Password,
+    DateTime Birthdate,
+    EmployeePosition Position
```

Comment:
this DTO is only used to register Admin or Managers, should we allow specifying other Positions?